tests/test_api.rs

```diff
@@ -10,6 +10,7 @@ use std::sync::Mutex;
 
 use rusty_v8 as v8;
 // TODO(piscisaureus): Ideally there would be no need to import this trait.
+use std::ffi::c_void;
```

Comment:
```suggestion
use std::ffi::c_void;
// TODO(piscisaureus): Ideally there would be no need to import this trait.
```
Otherwise the comment no longer makes sense.